src/v3/src/transformer/phone/phoneVerificationTransformer.ts

```diff
@@ -58,33 +58,14 @@ export const transformPhoneVerification: IdxStepTransformer = ({ transaction, fo
   // (Display order) 3rd element in the elements array
   uischema.elements.unshift(carrierInfoText);
 
-  const redactedPhoneNumber = nextStep.relatesTo?.value?.profile?.phoneNumber as string;
-  // using the &lrm; unicode mark to keep the phone number in ltr format
-  // https://www.w3.org/TR/WCAG20-TECHS/H34.html
-  const phoneNumberSpan = redactedPhoneNumber ? `<span class="strong no-translate">&lrm;${redactedPhoneNumber}</span>` : null;
-  const phoneInfoText = phoneNumberSpan || loc('oie.phone.alternate.title', 'login');
-  const smsInfoTextElement: DescriptionElement = {
-    type: 'Description',
-    contentType: 'subtitle',
-    options: {
-      content: `${loc('oie.phone.verify.sms.sendText', 'login')} ${phoneInfoText}`,
-    },
-  };
-  const voiceInfoTextElement: DescriptionElement = {
-    type: 'Description',
-    contentType: 'subtitle',
-    options: {
-      content: `${loc('oie.phone.verify.call.sendText', 'login')} ${phoneInfoText}`,
-    },
-  };
-
+  const subtitleElement = buildPhoneVerificationSubtitleElement(
+    nextStep.name,
+    primaryMethod as string,
```

Comment:
let's not cast when possible - either use a guard to assure the TS compiler that this will be defined or allow the possibility of `undefined`